frontend/src/main/java/puf/frisbee/frontend/view/LevelView.java

```diff
@@ -30,6 +30,9 @@
 
 	@FXML
 	private Button buttonLevelContinue;
+	
+	@FXML
+	private Label labelTeam;
```

Comment:
Alternativer Vorschlag: `labelTeamName` - außer wir können ausschließen, dass wir noch andere Team Properties anzeigen wollen 🤔

Reply:
Ja richtig, hatte ich so auch erst umgesetzt, allerdings dann im UML-Klassendiagramm noch einmal nachgeschaut und gesehen, dass eigentlich fast nichts aus dem TeamModel ins LevelModel wandert. Selbst die Daten zum Level und Score wergen im GameModel verwaltet. Aber ich lasse mich auch umstimmen, wenn's anders besser passt.

Reply:
Wärst Du damit einverstanden, dass wir dann bei allen Attributen das Präfix "team" ergänzen? Dann wären wir sehr klar und konsistent. OK?

Reply:
Ich bin verwirrt - Level ist doch das aktuelle Level, das gespielt wird? Und der Countdown ist ja auch unabhängig vom Team ... oder meinst du in der Highscore View? Da finde ich es gut :)

Reply:
Find ich gut.